lib/Gym/environment.ex

```diff
@@ -54,7 +52,7 @@ defmodule Gyx.Gym.Environment do
   end
 
   def handle_call({:make, environment_name}, _from, state) do
-    {env, initial_state} =
+    {env, initial_state, action_space} =
       HelperAsync.call(
```

Comment:
fuera del scope de este PR, pero probablemente hubiera nombrado al folder helper, y al modulo PythonAsync..  ahorita leo HelperAsync y no es fácil saber para que sirve sin irme a buscar el código.. Tal vez un nombre más descriptivo también.. PythonWrapper, PythonBridge, Python no see..

Reply:
The module name is `Gyx.Python.HelperAsync` , maybe after the alias this Python reference is lost

Reply:
Yes, I know and understand, but that's exactly the point.. in other projects I usually have a helpers folder in which I store functions I am going to be using all around the project, and not name the folder with a name for the function and then put a helper file inside.. 
For example, you could name it `Gyx.Helpers.Python`, and the alias would make this function call look like.. `Python.call`, and then you can keep adding things to that helpers folder if it makes sense.. 🤷🏼‍♂️